dev/io.openliberty.http.monitor/bnd.bnd

```diff
@@ -76,4 +77,6 @@ DynamicImport-Package: \
 	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
 	io.openliberty.jakarta.restfulWS.3.1,\
 	io.openliberty.jakarta.annotation.2.1;version=latest,\
```

Comment:
```suggestion
```

From what I can tell restfulWS 3.1 and annotations are used at all.  As such they should not be on the buildpath.

Reply:
fixed